histomicsui/web_client/templates/panels/metadataWidget.pug

```diff
@@ -0,0 +1,28 @@
+.s-panel-title-container
+  span.s-panel-title.h-panel-title
+    i.icon-tags
+      |  #{title}
```

Comment:
This should be outdented to the same level as the i tag above it:
```
    i.icon-tags
    |  #{title}
```

Reply:
Thank you, David. I have changed it.